internal/impl/babysitter.go

```diff
@@ -211,6 +228,10 @@ func (b *babysitter) watchPods(ctx context.Context, component string) error {
 					delete(addrs, pod.Name)
 					changed = true
 				}
+			case watch.Error:
+				errObject := apierrors.FromObject(event.Object)
+				logger.Error("watcher error", "err", errObject)
+				// do nothing else? do we need? even with retryWatcher?
```

Comment:
This is how RetryWatcher implements the error events... any ideas what we can do here? return an error and then re-create the watcher?